hudi-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadata.java

```diff
@@ -199,10 +203,14 @@ private void initIfNeeded() {
           }
         } else {
           // only log record
+          LOG.warn("TEST_LOG. 111 Looking for record " + key + ", isPresent in log " + logRecords.containsKey(key));
           if (logRecords.containsKey(key) && logRecords.get(key).isPresent()) {
+            LOG.warn("TEST_LOG. 222 found log record " + key);
             HoodieRecordPayload mergedPayload = logRecords.get(key).get().getData().preCombine(hoodieRecord.getData());
+            LOG.warn("TEST_LOG. 3333 merged log record " + key);
             result.add(Pair.of(key, Option.of(new HoodieRecord(hoodieRecord.getKey(), mergedPayload))));
           } else { // not found in both base file and log files
+            LOG.warn("TEST_LOG. 444 not log record found " + key);
```

Comment:
revert all this?

Reply:
yes, there is some test failure I am looking to triage. could not repro locally. will revert before landing the patch.